crates/nu-cli/src/commands/table.rs

```diff
@@ -74,13 +74,10 @@ pub fn from_list(values: &[Value], starting_idx: usize) -> nu_table::Table {
             _ => ansi_term::Color::Green,
         };
 
-        let header_bold = match config.get("header_bold") {
-            Some(b) => match b.as_bool() {
-                Ok(b) => b,
-                _ => true,
-            },
-            _ => true,
-        };
+        let header_bold = config
+            .get("header_bold")
+            .map(|x| x.is_true())
+            .unwrap_or(true);
```

Comment:
Dang, these are not equivalent.  I just realized that the `_ => true` from the original will result in a `false` in this format...